src/api/spec/controllers/webui/download_on_demand_controller_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+
+RSpec.describe Webui::DownloadOnDemandController do
+  let(:admin_user) { create(:admin_user) }
+  let(:repository) { create(:repository) }
+  let(:project)    { create(:project) }
+
+  let(:dod_parameters) {
```

Comment:
Well, a Hash object. Anything wrong with it?:-)

Basically it's all data needed for creating a DoD repo. I noticed I kept using the same set of data so I moved it into a 'let'. If yo know a better way of dealing with this please tell me:-)